docs/SubscriptionTestingGuide.md

```diff
@@ -0,0 +1,525 @@
+
+- Now you can see that DAO has subscribed to the basic tier:
+
+<img src="images/subscription/Screenshot%202023-06-29%20at%2018.07.56.png" width="600" style="padding-left: 50px;">
+
+- Observe that `fnBalance` at basic tier is set to 10K function calls - this number will decrease while we continue working with the DAO
+
+## Create Council Proposal
+
+- Go to `Developer --> Extrinsics` and select `daoCouncil --> propose` 
+
+- Use the account that you've created recently(0 SCTL tokens) as a tx submitter
+  
+- Set the following parameters for a call:
+
+```
+daoId: 0
+```
+
+- For the proposal call use the following:
+
+```
+proposal: daoTreasury --> transferToken
+daoId: 0
+amount: 1000
+beneficiary: select the wallet account you'be recently created
+lengthBound: 10000
+```
+
+<img src="images/subscription/Screenshot%202023-06-29%20at%2018.20.29.png" width="600" style="padding-left: 50px;">
+
+- `Submit transaction`
+
+- In the submission modal you can observe that `fees of 0` will be applied:
+
+<img src="images/subscription/Screenshot%202023-06-30%20at%2013.40.31.png" width="600" style="padding-left: 50px;">
+
+- Transaction successfully submitted and no gas fee has been charged since the account balance was 0 SCTL
+
+- Please note that DAO calls are also limited by the `fnPerBlockLimit` for now so DAO members can make at most 100 function calls per block as per the current settings
+
+- Go to `Developer --> Chain State`
+  
+- Select `daoSubscription --> subscriptions` and press `+` button
+
+- You can observe that `fnBalance` has decrease by 1:
+
+<img src="images/subscription/Screenshot%202023-06-30%20at%2013.33.23.png" width="600" style="padding-left: 50px;">
+
+- You can try to create a similar proposal by the 2nd account that you've created earlier receiving the following error:
+
+<img src="images/subscription/Screenshot%202023-07-03%20at%2016.12.05.png" width="400" style="padding-left: 50px;">
+
+- It is custom runtime tx error [Not DAO Member](../primitives/dao/src/lib.rs#L643)
+
+## Vote for Council Proposal
+
+- Go to `Developer --> Chain State`
+  
+- Select `daoCouncil --> proposals` and press `+` button
+
+- Copy the proposal hash that will be used for voting
+
+- Open `Developer --> Extrinsics` section
+
+- From the list of available extrinsics select `daoCouncil --> vote` callable
+
+- Use the following parameters for the voting call:
+
+```
+account: Alice
+daoId: 0
+proposal: 0x5ed15c6ac27c4bd22a5f084e4be38af2904f2e669abf5b35e7f97975aeefcbf0
+index: 0
+approve: Yes
+```
+
+<img src="images/subscription/Screenshot%202023-07-03%20at%2016.32.45.png" width="600" style="padding-left: 50px;">
+
+- `Submit Transaction`
+
+- repeat the voting step above using the 1st account you've created earlier
+
+- Select `daoCouncil --> close` callable
+
+- Use the following parameters for the voting close call:
+
+```
+account: 1st account from the wallet
+daoId: 0
+proposal: 0x5ed15c6ac27c4bd22a5f084e4be38af2904f2e669abf5b35e7f97975aeefcbf0
```

Comment:
Same thing here. Add note proposal hash will be different for user.

Reply:
same as above